### DISCUSSION TITLE
Possible grammar error in HTML accessibility guide?

### DISCUSSION THREAD

**Discussion by jksprat** [mdn/mdn-community#659] [Content] [answered]
In the HTML accessibility guide, in the second to last paragraph within the [Page Layouts](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML#page_layouts) section, the following is written:

> If you try our more modern structure example with a screen reader, you'll see that the layout markup no longer gets in the way and _confuses the content readout._

Did the writer mean to say something along the lines of "**_no longer confuses_** the content readout" or "**_doesn't confuse_** the content readout"?

**Comment by dipikabh** [comment]
Hi @jksprat, thanks for raising the concern about the wording. Yes, the intention is to say "does not confuse".
We can change the "and" to "or" like so: "If you try our more modern structure example with a screen reader, you'll see that the layout markup no longer gets in the way or cause confusion with the content readout." 

Would you like to open a pull request to submit this fix?

**Reply by jksprat** [reply]
That sounds fine, though it should be "causes", not "cause".

I'm not familiar with how github works so I don't know how to do that.

**Reply by dipikabh** [reply, marked as answer]
I've opened https://github.com/mdn/content/pull/32997.
